File: fizz-buzz/fizz-buzz.cs
public class Solution {
    public IList<string> FizzBuzz(int n) {
        IList<string> l = new List<string>(); 
        for (int i = 1; i <= n; i++) {
            string str = "";
            if (i % 3 == 0) str += "Fizz";
            if (i % 5 == 0) str += "Buzz";
            l.Add(str != "" ? str : i.ToString());
        }
        return l;
    }
}

File: fizz-buzz/fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> l = new ArrayList<String>(); 
        for (int i = 1; i <= n; i++) {
            String str = "";
            if (i % 3 == 0) str += "Fizz";
            if (i % 5 == 0) str += "Buzz";
            l.add(str != "" ? str : Integer.toString(i));
        }
        return l;
    }
}